test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -104,6 +120,7 @@ def hdfs2Jar = tasks.register("hdfs2jar", ShadowJar) {
   }
   archiveClassifier.set("hdfs2")
   from sourceSets.main.output
+  from patchTask
```

Comment:
Should we also exclude the non-patched classes here?
The generated JAR is OK, it contains the patched classes and I see no error, but I don't know if we need to be more explicit here.